src/test/java/com/uid2/shared/middleware/AttestationMiddlewareTest.java

```diff
@@ -65,8 +64,9 @@ void setUp() {
     @Test
     void trustedValidJwtNoRolesReturnsSuccess() throws JwtService.ValidationException {
         var attestationMiddleware = getAttestationMiddleware(true);
-        JwtValidationResponse response = new JwtValidationResponse(true).withSubject(hashedKey);
-        when(this.jwtService.validateJwt("dummy jwt", this.jwtAudience, this.jwtIssuer)).thenReturn(response);
+        JwtValidationResponse response = new JwtValidationResponse(true)
+                .withSubject(EXPECTED_OPERATOR_KEY_HASH);

```

Comment:
I'm guessing this test is passing because, as we learned, the subject validation is skipped?